os/kernel/mqueue/mq_timedreceive.c

```diff
@@ -230,7 +230,7 @@ ssize_t mq_timedreceive(mqd_t mqdes, FAR char *msg, size_t msglen, FAR int *prio
 
 	rtcb->waitdog = wd_create();
 	if (!rtcb->waitdog) {
-		set_errno(EINVAL);
+		set_errno(ENOMEM);
```

Comment:
Please add description of ENOMEM in the function description.

Reply:
Added. Thanks.